frontend/notarial-blockchain-ui/src/App.tsx

```diff
@@ -1,28 +1,152 @@
-import './App.css';
-import Button from '@mui/material/Button';
+import "./App.css";
 
-const  App = () => {
+import { CircularProgress } from "@material-ui/core";
+import Button from "@mui/material/Button";
+import axios from "axios";
+import React, { useCallback, useState } from "react";
+
+type ConnectionType = "NOT_CONNECTED" | "CONNECTING_ERROR" | "CONNECTED";
+type User = {
```

Comment:
shoudln't that be located in some "model" directory?

Reply:
in the future, when we will split the project into components, now it's simple one page to help develop REST